backend/rates/history.go

```diff
@@ -278,38 +278,36 @@ func (updater *RateUpdater) fetchGeckoMarketRange(ctx context.Context, coin, fia
 	}
 
 	// Make the call, abiding the upstream rate limits.
-	msg := fmt.Sprintf("fetch coingecko coin=%s fiat=%s start=%s", coin, fiat, timeRange.start)
+	//	msg := fmt.Sprintf("fetch coingecko coin=%s fiat=%s start=%s", coin, fiat, timeRange.start)
```

Comment:
undo?

Reply:
Hey yeah, the issue is that this was passed to a fn call that doesn't exist anymore, I think for logging purposes and this is a bit harder now to achieve. I was planning to find a way around this and only then open for review :)

Reply:
I don't think there is a way here to have the log message show up periodically as we did before. `Wait()` is a blocking call  and we have no way of loggin periodically as we did before.